sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -21,7 +21,6 @@ import java.net.URI
 
 import org.apache.spark.sql._
 import org.apache.spark.sql.catalyst.catalog._
-import org.apache.spark.sql.catalyst.expressions.Attribute
```

Comment:
If possible, let's not touch this because we didn't change anything in this file. It would be helpful for backporting.
SPARK-25271 is reported as a regression in 2.3.x. I assume that we need to backport this for 2.4.1 and 2.3.3 at least.